### DISCUSSION TITLE
Feature Request: Ability to choose how many variables to change on each trial

### DISCUSSION THREAD

**Discussion by ea2399** [akumidv/tradingview-assistant-chrome-extension#122] [Ideas]
Basically allowing the ability to choose how many variables the extension will change on each trial without changing all of them at once like when using Random.

**Comment by akumidv** [comment]
You already can do this by checkbox before parameters 
![image](https://github.com/user-attachments/assets/42b862af-7748-45ac-a4d5-d81de4badc7d)
